app/controllers/api/v1/rooms_controller.rb

```diff
@@ -30,6 +30,7 @@ def index
         end
 
         rooms = user_rooms + shared_rooms
+        rooms.sort_by { |room| room.online ? 0 : 1 }
```

Comment:
Shouldn't we do this after calling the RunningMeeting service?

Reply:
We'll need to find a better way to handle the sorting and the queries. I'll spend some time later cleaning it up

Reply:
Right.

Reply:
Should we sort via the frontend as it was initially?